src/Serialization/ParseNodeFromStringTrait.php

```diff
@@ -14,7 +14,7 @@ trait ParseNodeFromStringTrait
     /**
      * @throws Exception
      */
-    public function parseDateIntervalFromString(string $value): DateInterval
+    private function parseDateIntervalFromString(string $value): DateInterval
```

Comment:
will you be able to use the method in the serialization packages if you do that?

Reply:
Yes, and in that case, we won't have to specify visibility for each method unless we want to expose them outside the class.